perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -411,7 +416,7 @@ def Prepare(benchmark_spec):
   logging.info('Umount scratch disk on %s at %s', vm, mount_point)
   vm.RemoteCommand('sudo umount %s' % mount_point)
 
-  if FLAGS.device_fill_size is not '0':
+  if FLAGS.prefill_device:
```

Comment:
I believe this should be:

if FLAGS.against_device or FLAGS.prefill_device:

That is, if the user wants to test against device, then we will always prefill the device (igoring the FLAGs.prefill_device); however, if the user wants to test using file system, then we read the prefill_device flag to check if the user still wants to fill the device even when they are testing against the file system.

Without my proposed fix, any caller who sets against_device to true would also *always* set prefill_device to true which is redundant. 

Also all our current automation that runs FIO tests against device, they would be broken after this code (because existing automations aren't aware of the prefill_device flag)